RecoTauTag/RecoTau/plugins/PFRecoTauDiscriminationByHPSSelection.cc

```diff
@@ -258,10 +256,13 @@ PFRecoTauDiscriminationByHPSSelection::discriminate(const reco::PFTauRef& tau) c
   }
   // Now check the pizeros
   BOOST_FOREACH(const reco::RecoTauPiZero& cand, tau->signalPiZeroCandidates()) {
+    double dEta = TMath::Max(0., fabs(cand.eta() - tauP4.eta()) - cand.bendCorrEta());
+    double dPhi = TMath::Max(0., fabs(cand.phi() - tauP4.phi()) - cand.bendCorrPhi());
```

Comment:
is your correction always one-sided (always reducing the |deltaPhi| )?
``` reco::deltaPhi(cand.phi() - tauP4.phi() ) ``` should be used instead of a plain delta to avoid +/-pi wrap-around problem